js/repl/Repl.js

```diff
@@ -98,18 +104,23 @@ class Repl extends React.Component {
     // A partial State is defined first b'c this._compile needs it.
     // The compile helper will then populate the missing State values.
     this.state = {
-      babel: persistedStateToBabelState(persistedState),
-      builtIns: persistedState.builtIns,
+      babel: persistedStateToBabelState(persistedState, babelConfig),
       code: persistedState.code,
       compiled: null,
       compileErrorMessage: null,
       debugEnvPreset: persistedState.debug,
       envConfig,
       envPresetDebugInfo: null,
-      envPresetState: configToState(
+      envPresetState: persistedStateToEnvState(
```

Comment:
Not a blocker, but we should step back and re-think/clean up the state shape now that we have all a number of things that may or may not be loaded _and_ depending on version, etc